rocAL/include/api/rocal_api_info.h

```diff
@@ -129,4 +129,12 @@ extern "C" const char* ROCAL_API_CALL rocalGetErrorMessage(RocalContext rocal_co
  */
 extern "C" TimingInfo ROCAL_API_CALL rocalGetTimingInfo(RocalContext rocal_context);
 
+/*!
+ * \brief Retrieves the information about the size of the last batch.
+ * \ingroup group_rocal_info
+ * \param rocal_context
+ * \return The number of samples that were padded in adherence with last_batch_policy and last_batch_padded
```

Comment:
Make it -> The number of samples that were padded in the last batch in adherence with last_batch_policy and last_batch_padded